tests/js/fixtures/automations.ts

```diff
@@ -0,0 +1,15 @@
+import {DataConditionGroupFixture} from 'sentry-fixture/dataConditions';
+
+import type {Automation} from 'sentry/types/workflowEngine/automations';
+
+export function AutomationFixture(params: Partial<Automation>): Automation {
+  return {
+    id: '1',
+    name: 'Automation',
+    lastTriggered: new Date('2025-01-01T00:00:00.000Z'),
```

Comment:
should be a string coming from the api
```suggestion
    lastTriggered: '2025-01-01T00:00:00.000Z',
```

Reply:
Fixed in f5262caffa9d406e4de631e62993f59c4fae35e9 ✨